src/main/java/com/epam/test_generator/entities/api/CaseTrait.java

```diff
@@ -0,0 +1,24 @@
+package com.epam.test_generator.entities.api;
+
+import com.epam.test_generator.entities.Step;
+import com.epam.test_generator.services.exceptions.BadRequestException;
+import java.util.List;
+import java.util.Objects;
+
+public interface CaseTrait {
```

Comment:
> CaseTrait an interface that describes Case's behavior (add/remove the case, checking for Case belonging to Suit)
You mentioned that in the PR desc. But I don't see any methods for adding/removing case.